chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -244,6 +244,7 @@
   protected boolean waterPlaneEnabled = false;
   protected double waterPlaneHeight = World.SEA_LEVEL;
   protected boolean waterPlaneOffsetEnabled = true;
+  protected boolean waterPlaneClip = true;
```

Comment:
I would maybe rename it to waterPlaneChunkClip to make it clear what is getting clipped.